release-handbook/major_release_workflow.md

```diff
@@ -0,0 +1,184 @@
+
+Major Zowe releases follow a similar publication process to that of minor releases: content is staged, and then that content is published to the live Zowe Docs site.
+
+What is quite different about updating documentation for a major release is the amount and type of prep work required. To improve the chances of a smoother publication process, you will want to start preparing the doc work for a major release at least a year ahead of the major release GA date.
+
+Zowe major releases are typically done about every two years or so. Minor releases take place about once a calendar quarter.
+
+The publication process for minor releases is documented in the [Release handbook](release_handbook.md).
+
+For major Zowe releases, follow the process outlined here.
+
+## Major release publication process
+
+1. At least a year before the major release GA, Doc Squad works together to create a release strategy.
+    - Strategy to include:
+        - Schedule for reviewing existing content for any needed modifications
+        - Schedule for when modifications should be drafted, reviewed, and completed
+        - Complete inventory of Zowe Docs content and the people who are responsible for ensuring that content is ready for the major release.
+        - Assignment of responsibilities.
+2. Create a new branch based off the `master` branch to be dedicated to the major release documentation. Open the new branch in Visual Studio Code.
+    - For example, `v3-doc-branch` was used for the Zowe V3 release.
+3. Archive the documentation for the most recent (minor) release by taking the following steps in this order:
+    - In the Terminal, run `npm run docusaurus docs:version <version>`.
+
+        - *\<version>* is the version number of the previous release.
+
+            Example: At the time you create the new branch, Zowe 3.5 is the latest version for the current major release. That means the command must be issued as `npm run docusaurus docs:version v3.5.x`
+
+        - When the command completes, it returns the message `[docs]: version v3.5.x created!`
+
+        - The document versioning mechanism has:
+            * Copied the full `docs/` folder contents into a new `versioned_docs/version-v3.5.x/` folder.
+            * Created a versioned `sidebars` file based from your current sidebar configuration. The file is saved as `versioned_sidebars/version-v3.5.x-sidebars.json`.
+            * Appended the new version number to the `versions.json` file.
+
+    - Archive the `/static` files:
+        1. Go to the `/static` folder.
+        2. Create an empty directory with the name of the previous version in `/static`. For example: `static/v3.5.x`.
+        3. Copy all contents of the `/static/stable` directory and paste them in the previous version's empty directory in the previous step. For example: `/static/v3.5.x`.
+
+    - Update a set of links in the archived documentation to ensure that they refer to the correct location:
```

Comment:
Is there a way to run the link checker locally? 
That way we don't have to rely on manual link-verification 😋

Reply:
It can be run locally, but would require installing extra deps like Rust/cargo.
```shell
cargo install lychee
lychee --accept 403,406,429 --cache --fallback-extensions md --include-fragments --max-cache-age 1d --max-concurrency 2 --timeout 60 --verbose 'docs/**/*.md'
```